### DISCUSSION TITLE
Connecting to a Slingboxes-M1

### DISCUSSION THREAD

**Discussion by kasibarndoor** [GerryDazoo/Slinger#530] [General]
I have read that if a hard reset is applied to an M1 that access is gained by slinger with the admin password admin, I would like to connect to the Slingbox and confirm that the reset has been successful. 

If I connect with SSH what userid password and port do I use?

**Comment by regmar505** [comment]
Do you mean to ssh to the slingbox? Not possible, ssh user/root password is unknown. The admin password you talked about is to connect to the slingbox to initiate a stream.
